src/components/Dashboard/Users.js

```diff
@@ -7,63 +7,139 @@ import {
   DashboardLayout as Layout
 } from 'components/Dashboard/';
 import { bindActionCreators } from 'redux';
-import { Table } from 'react-bootstrap';
+import { Col, Container, Row, Table } from 'react-bootstrap';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 import { actions as usersActions } from 'reducers/users';
-import { getAllUsers } from 'selectors/users';
+import { getAllUsers, getUsersPager } from 'selectors/users';
 
 export class Users extends Component {
   static propTypes = {
     actions: PropTypes.object.isRequired,
-    users: PropTypes.array,
-    layoutOptions: PropTypes.object
+    layoutOptions: PropTypes.object,
+    collection: PropTypes.array,
+    pager: PropTypes.object
   };
 
+  constructor(props) {
+    super(props);
+    this.state = { limit: 20, page: 1 };
+    this.handlePageChange = this.changePage.bind(this);
+    this.handleLimitChange = this.changeLimit.bind(this);
+    this.handleLimitUpdate = this.updateLimit.bind(this);
+  }
+
   componentDidMount() {
     const { actions } = this.props;
 
-    actions.requestUsers();
+    actions.requestUsers({ page: 1, limit: 20 });
+  }
+
+  pagerCounter() {
+    const { pages } = this.props.pager;
+
+    let i = 1;
+
+    const pager = [];
```

Comment:
I think part of the reason this PR is so large is because there is a lot of duplicated functionality in these components. You could define an [HOC](https://reactjs.org/docs/higher-order-components.html) called e.g. `withPagination` that adds these functions (pagerCounter, changePage, changeLimit, updateLimit) to the wrapped component, and I think it would cut a few thousand lines out of the PR.